src/action.rs

```diff
@@ -10,9 +10,11 @@ use crate::{authority::Authority, error::InterfaceError, xor_space::XorName};
 use bytes::Bytes;
 use hex_fmt::HexFmt;
 use quic_p2p::Token;
```

Comment:
Same problem as other PR.
This commit was already merged:
- routing/refactor deny use of depricated methods

This commit should not be there:
- Merge branch 'fleming' of github.com:maidsafe/routing into fleming 

Is there something strange with your local fleming branch?

Reply:
Not sure @jeanphilippeD  my fleming branch is updated. I did branch from this branch to the other PR (large_enum) and had to reverse cherry-pick some commits. It all worked out in the end but a few hours of git madness was my Saturday fun. I eventually got both PR's separated out in the end. Git madness basically